pyrate/core/phase_closure/closure_check.py

```diff
@@ -145,11 +146,13 @@ def filter_to_closure_checked_ifgs(config, interactive_plot=True) -> \
         if len(ifg_files) == len(new_ifg_files):
             break
         else:
+            i += 1
             ifg_files = new_ifg_files  # exit condition could be some other check like number_of_loops
+            # TODO: update params with new ifg_files here
```

Comment:
@basaks. Most functions are reading the `ifg_files` from the `params`. So this needs to be updated here following each iteration.